docs/source/configuration/volto-slate/configuration-settings.md

```diff
@@ -239,6 +239,15 @@ They are not persisted in the final value, so they are useful, for example, to h
 slate.runtimeDecorators = [([node, path], ranges) => ranges];
 ```
 
+(editor-configuration-slate-useLinkedHeadlines-label)=
+
+## `slate.useLinkedHeadlines`
+
+volto-slate comes with anchored headings for all tags(h1,h2,h3,h4) formatted via editor. You can easily opt-out by using the setting.
```

Comment:
```suggestion
The setting `slate.useLinkedHeadlines` controls whether `volto-slate` creates anchors for headings, such as `h1` and `h2`, in the editor.
The default setting is `true`.
You can opt out of this feature by setting its value to `false`.
```

Also possibly refactor the code and code sample, per previous comment.

Reply:
Thanks @stevepiercy . I think "linked headings" would be much more precise than headlines.

Reply:
Fixed in [de9e57f](https://github.com/plone/volto/pull/4287/commits/de9e57f3797e705a451351046a9688722a28500c)